2021-10-13 00:01:21,567 INFO    MainThread:23024 [wandb_setup.py:_flush():69] setting env: {}
2021-10-13 00:01:21,568 INFO    MainThread:23024 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-13 00:01:21,568 INFO    MainThread:23024 [wandb_init.py:_log_setup():342] Logging user logs to f:\Lab\nfs\base-visob\wandb\run-20211013_000121-112g3upd\logs\debug.log
2021-10-13 00:01:21,568 INFO    MainThread:23024 [wandb_init.py:_log_setup():343] Logging internal logs to f:\Lab\nfs\base-visob\wandb\run-20211013_000121-112g3upd\logs\debug-internal.log
2021-10-13 00:01:21,568 INFO    MainThread:23024 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001F02B4FD5E0>
2021-10-13 00:01:21,569 INFO    MainThread:23024 [wandb_init.py:init():375] calling init triggers
2021-10-13 00:01:21,569 INFO    MainThread:23024 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-10-13 00:01:21,569 INFO    MainThread:23024 [wandb_init.py:init():395] re-initializing run, found existing run on stack: 1kqv6emi
2021-10-13 00:01:21,570 INFO    MainThread:23024 [wandb_run.py:finish():1271] finishing run sreeraj/UFPR-Ocular/1kqv6emi
2021-10-13 00:01:21,570 INFO    MainThread:23024 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-10-13 00:01:21,571 INFO    MainThread:23024 [wandb_run.py:_restore():1543] restore
2021-10-13 00:01:22,518 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 17405
  total_bytes: 17405
}

2021-10-13 00:01:22,686 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 17405
  total_bytes: 17405
}

2021-10-13 00:01:22,831 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 17405
  total_bytes: 17405
}

2021-10-13 00:01:23,657 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 17405
  total_bytes: 23589
}

2021-10-13 00:01:23,762 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 17405
  total_bytes: 23589
}

2021-10-13 00:01:23,873 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 23589
  total_bytes: 23589
}

2021-10-13 00:01:23,983 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 23589
  total_bytes: 23589
}

2021-10-13 00:01:24,095 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 23589
  total_bytes: 23589
}

2021-10-13 00:01:24,203 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 23589
  total_bytes: 23589
}

2021-10-13 00:01:24,314 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 23589
  total_bytes: 23589
}

2021-10-13 00:01:24,451 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 23589
  total_bytes: 23589
}

2021-10-13 00:01:24,562 INFO    MainThread:23024 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
  media_count: 1
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 23589
  total_bytes: 23589
}

2021-10-13 00:01:26,032 INFO    MainThread:23024 [wandb_run.py:_show_summary():1848] rendering summary
2021-10-13 00:01:26,033 INFO    MainThread:23024 [wandb_run.py:_show_history():1886] rendering history
2021-10-13 00:01:26,035 INFO    MainThread:23024 [wandb_run.py:_show_files():1915] logging synced files
2021-10-13 00:01:26,045 INFO    MainThread:23024 [wandb_init.py:init():424] starting backend
2021-10-13 00:01:26,045 INFO    MainThread:23024 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-10-13 00:01:26,048 INFO    MainThread:23024 [backend.py:ensure_launched():135] starting backend process...
2021-10-13 00:01:26,197 INFO    MainThread:23024 [backend.py:ensure_launched():139] started backend process with pid: 30240
2021-10-13 00:01:26,197 INFO    MainThread:23024 [wandb_init.py:init():429] backend started and connected
2021-10-13 00:01:26,198 INFO    MainThread:23024 [wandb_run.py:_label_probe_notebook():815] probe notebook
2021-10-13 00:01:26,198 INFO    MainThread:23024 [wandb_run.py:_label_probe_notebook():822] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-10-13 00:01:26,199 INFO    MainThread:23024 [wandb_init.py:init():477] updated telemetry
2021-10-13 00:01:26,199 INFO    MainThread:23024 [wandb_init.py:init():500] communicating current version
2021-10-13 00:01:27,937 INFO    MainThread:23024 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-13 00:01:27,938 INFO    MainThread:23024 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-13 00:01:28,189 INFO    MainThread:23024 [wandb_init.py:init():540] starting run threads in backend
2021-10-13 00:01:31,864 INFO    MainThread:23024 [wandb_run.py:_console_start():1601] atexit reg
2021-10-13 00:01:31,865 INFO    MainThread:23024 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-10-13 00:01:31,865 INFO    MainThread:23024 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-10-13 00:01:31,866 INFO    MainThread:23024 [wandb_run.py:_redirect():1536] Redirects installed.
2021-10-13 00:01:31,866 INFO    MainThread:23024 [wandb_init.py:init():565] run started, returning control to user process
2021-10-13 00:01:32,200 INFO    MainThread:23024 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-13 00:01:32,200 INFO    MainThread:23024 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-10-13 00:08:11,651 INFO    MainThread:23024 [wandb_init.py:_resume_backend():265] resuming backend
2021-10-13 00:08:11,663 INFO    MainThread:23024 [wandb_setup.py:_flush():69] setting env: {}
2021-10-13 00:08:11,664 INFO    MainThread:23024 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-13 00:08:11,664 INFO    MainThread:23024 [wandb_init.py:_log_setup():342] Logging user logs to f:\Lab\nfs\base-visob\wandb\run-20211013_000811-11i3hrvp\logs\debug.log
2021-10-13 00:08:11,664 INFO    MainThread:23024 [wandb_init.py:_log_setup():343] Logging internal logs to f:\Lab\nfs\base-visob\wandb\run-20211013_000811-11i3hrvp\logs\debug-internal.log
2021-10-13 00:08:11,665 INFO    MainThread:23024 [wandb_init.py:init():375] calling init triggers
2021-10-13 00:08:11,666 INFO    MainThread:23024 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-10-13 00:08:11,666 INFO    MainThread:23024 [wandb_init.py:init():395] re-initializing run, found existing run on stack: 112g3upd
2021-10-13 00:08:11,668 INFO    MainThread:23024 [wandb_run.py:finish():1271] finishing run sreeraj/UFPR-Ocular/112g3upd
